ch2-git-basics/cloning.qmd

```diff
@@ -0,0 +1,57 @@
+## Cloning an Existing Repository
+
+### Introduction
+
+This section will guide you through the process of cloning a GitHub repository. Cloning a repository means making a local copy of a project that exists on GitHub. This enables you to make changes and test locally before pushing to be approved and merged with the main branch on GitHub.
+
+There are several ways to clone an existing repository. There are two methods detailed below. The first uses Rstudio and the second uses Git Bash. Start with step 1, which is the same for both, and then decide which method to follow. Once you have successfully cloned, you should see the files appear in both the file you have chosen to store your repository in and the files pane in R studio.
+
+### Step 1: Clone the Repository
+
+a\. Go to the [git-academy-sandbox](https://github.com/dfe-analytical-services/git-academy-sandbox) repository page on GitHub.
+
+b\. Click the "Code" green button and copy the URL (HTTPS link).
+
+### To Clone Using RStudio:
```

Comment:
How about we try using a tabset panel for the different methods?

https://quarto.org/docs/interactive/layout.html#tabset-panel